src/GraphQL/Resolvers/AsyncFieldResolver.cs

```diff
@@ -24,6 +28,7 @@ object IFieldResolver.Resolve(IResolveFieldContext context)
         }
     }
 
+    /// <inheritdoc cref="AsyncFieldResolver{TReturnType}"/>
```

Comment:
Would you like to say a few words about how this resolver differs from the previous one?

Reply:
Sure